Implement "tc" (change token text) command in graph editor (#378)
hexatomic/hexatomic: docs/user/src/usage/graph-editor/console.md

ta #t1 is a
```

## Change the text of a token: `tc`

> Change which text an existing token represents.

Reviewer: ```suggestion
Change the text that an existing token covers.
```

We're not changing the covered text, but change the text itself. This wording represents this better I think. Also changed to "covers", as this again appears further below, and the concept should be clear to users at this stage (or can be either looked up in the docs or inferred).

### Examples

> Starting with an initial text with the two tokens `[This] [text]` (first one is called "t1" and the second one "t2"),

Reviewer: ```suggestion
Let's start with an initial text with the two tokens `[This] [text]`. The first token is called "t1" and the second one "t2".
```
Make this easier to parse.

> executing

Reviewer: ```suggestion
You can execute the following command:
```

Again easier to parse, further breaking up the sentence.

```text
tc #t2 example
```

> will change the covered token text from "text" to "example": `[This] [example]`.

Reviewer: ```suggestion
This will change the covered token text from "text" to "example": `[This] [example]`.
```

Ongoing effort to make this easier to parse.

> All token before and after are not affected and only one token can be changed at a time

Reviewer: ```suggestion
All tokens before and after the changed token are not affected.
Only one token can be changed at a time.
```
Fix typos and split sentence.
